elements/pfe-sass/extends/_typography-deprecated.scss

```diff
@@ -0,0 +1,39 @@
+// DEPRECATED 
+%pfe-headline-xxl  {
+    font-family: pfe-var(font-family--heading);
```

Comment:
note that these deprecated classes extend "heading" as opposed to "headline" vars